src/ocl/convolutionocl.cpp

```diff
@@ -192,6 +194,12 @@ EvaluateDataImplicitGemmSolution(Handle& handle,
 
     elapsed = 0.0f;
 
+    if(kernel.GetName().find("igemm_v4r1_dynamic") == 0) // NOLINT
```

Comment:
We should disable all the abseil checks for clang-tidy(since we are not using that library) instead of using NOLINT for this. Just need to add `-abseil-*` to the list of checks here:

https://github.com/ROCmSoftwarePlatform/MIOpen/blob/develop/CMakeLists.txt#L301

Reply:
OK, let me add it